virtcontainers/qemu.go

```diff
@@ -559,19 +559,6 @@ func (q *qemu) addDeviceToBridge(ID string) (string, string, error) {
 	return "", "", err
 }
 
-func (q *qemu) removeDeviceFromBridge(ID string) error {
```

Comment:
ok, that's weird, if AddDeviceToBridge is called [here](https://github.com/jodh-intel/runtimes/blob/734c387f4ed7bbcb993c5029ec5266468b1cdb4f/virtcontainers/qemu.go#L585) , this function MUST BE called when the device is removed , seems like something is broken

Reply:
/cc @amshinde, @sameo.

Reply:
@devimc that's a good point, we should be unplugging the devices from the bridge.